DAO.sol

```diff
@@ -607,52 +610,40 @@ contract DAO is DAOInterface, Token, TokenCreation {
             sumOfProposalDeposits -= p.proposalDeposit;
         p.open = false;
     }
-	
-    function soloSplit(
+
+    function withdraw(
         uint _proposalID,
         address _newCurator
     ) noEther onlyTokenholders returns (bool _success) {
-		
-		// Did you already vote on another proposal?
+
+        unblockMe();
+
+        // Did you already vote on another proposal?
         if (blocked[msg.sender] != _proposalID && blocked[msg.sender] != 0)  {
```

Comment:
> Not sure what this check does 'blocked[msg.sender] != _proposalID. Can the caller simply find out what proposalID is blocking them and call withdraw() with that proposal id which would allow them to bypass this check?

This `(blocked[msg.sender] != _proposalID && blocked[msg.sender] != 0)` essentially stops the caller from splitting if they are blocked in any proposal. So the check is also in the beginning of splitDao already.